README.md

```diff
@@ -1,49 +1,180 @@
-# Aviator Command Line Tool
-Aviator CLI is an open-source CLI tool to create, update, review and merge stacked PRs on GitHub.
+<div align="center">
+  <img src="resources/isotype-color.svg" width="100">
+  <p><strong>av:</strong> Stacked PR manager</p>
+</div>
 
-## What are Stacked PRs
-Stacked pull requests make smaller, iterative changes and are stacked on top of each other instead of bundling large monolith changes in a single pull request. Each PR in the stack focuses on one logical change only, making the review process more manageable and less time-consuming.
+---
 
-Read more about stacked PRs in our blog: [Rethinking code reviews with stacked PRs](https://www.aviator.co/blog/rethinking-code-reviews-with-stacked-prs/).
+`av` is a command-line tool that helps you manage your PRs in a stacked way. It
+allows you to create a PR that depends on another PR, and it will automatically
+update the dependent PR when the base PR is updated.
```

Comment:
```suggestion
allows you to create a PR stacked on top of another PR, and it will
automatically update the dependent PR when the base PR is updated.
```